### PR TITLE
Adding time to process for documents

### PR DESCRIPTION
**Pull request by alexjoybc**
# Description

This PR includes the following proposed change(s):

- [FLA-855](http://justice.gov.bc.ca/jira/browse/FLA-855)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [x] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/efiling-reviewer-api/src/main/java/ca/bc/gov/open/jag/efilingreviewerapi/document/DocumentsApiDelegateImpl.java [hunk 1]
```diff
@@ -81,6 +83,18 @@ public DocumentsApiDelegateImpl(DiligenService diligenService, ExtractRequestMap
     @Override
     public ResponseEntity<Void> documentEvent(UUID xTransactionId, DocumentEvent documentEvent) {
 
+
+        Optional<ExtractRequest> extractRequestCached = extractStore.get(documentEvent.getDocumentId());
+
+        if(extractRequestCached.isPresent()) {
```
Comment (TayGov) [inline src/backend/efiling-reviewer-api/src/main/java/ca/bc/gov/open/jag/efilingreviewerapi/document/DocumentsApiDelegateImpl.java:89]: This should be when we get processed status.

**Review by TayGov — CHANGES_REQUESTED** [review]
Just the one thing

**Review by TayGov — DISMISSED** [review]

File: src/backend/efiling-reviewer-api/src/main/java/ca/bc/gov/open/jag/efilingreviewerapi/extract/models/ExtractRequest.java [hunk 2]
```diff
@@ -7,39 +7,64 @@
...
     }
 
+    public void updateProcessedTimeMillis() {
+        this.processedTimeMillis = System.currentTimeMillis();
+    }
+
+    public long getProcessingTimeMillis() {
+
+        if(processedTimeMillis == null || receivedTimeMillis == null) {
+            return -1l;
```
Comment (akroon3r) [inline src/backend/efiling-reviewer-api/src/main/java/ca/bc/gov/open/jag/efilingreviewerapi/extract/models/ExtractRequest.java:36]: magic number?

Comment (TayGov) [reply src/backend/efiling-reviewer-api/src/main/java/ca/bc/gov/open/jag/efilingreviewerapi/extract/models/ExtractRequest.java:36]: That is long syntax. Add an L to the end.

**Review by karthikmsqa — APPROVED** [review]
LGTM thanks @TayGov

**Pull request merged**